.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Qlik Platform Examples contributing guide
+
+New platform examples or improvements to existing examples are always welcome. 
+
+- [Code of conduct](#code-of-conduct)
+- [Bugs](#bugs)
+- [Proposing a change](#features)
+- [Developing](#developing)
+- [Git guidelines](#git)
+- [Signing the CLA](#cla)
+
+## <a name="code-of-conduct"></a> Code of conduct
+
+Please read and follow our [Code of conduct](https://github.com/qlik-oss/open-source/blob/master/CODE_OF_CONDUCT.md)
+
+## <a name="bugs"></a> Bugs
+
+Bugs can be reported by filing a [new bug issue](https://github.com/qlik-oss/qlik-platform-examples/issues/new?template=bug.md) in the repository. Please make sure to browse through existing [issues](https://github.com/qlik-oss/qlik-platform-examples/labels/bug) before creating a new one.
+
+## <a name="features"></a> Proposing a change
+
+If you want to propose changes to this project, let us know by [filing a issue](https://github.com/qlik-oss/qlik-platform-examples/issues/new/choose).
+
+## <a name="git"></a> Git Guidelines
+
+Generally, development should be done directly towards the `main` branch.
+
+### Branching
```

Comment:
in some cases, ex: test in CI that requires secrets - this branching strategy is not ideal...but for the scripts, It should be fine

Reply:
Is there a better way to do this that you can point me at? We don't have secrets now, but we will at some point...